src/libmongoc/tests/test-mongoc-client-side-encryption.c

```diff
@@ -1089,13 +1089,15 @@ _test_key_vault (bool with_external_key_vault)
    mongoc_client_encryption_encrypt_opts_destroy (encrypt_opts);
 }
 
+/* Prose Test 3: External Key Vault Test */
```

Comment:
Thanks for going back and adding these. I forgot there were so many existing CSFLE prose tests.